data/schemas/ca.bc.gov.data.datasource.local.MyHealthDataBase/13.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
   "database": {
     "version": 13,
```

Comment:
Since develop is already on version 13, please upgrade db version to 14

Reply:
done